cluster/cluster_members.go

```diff
@@ -132,6 +132,22 @@ CREATE TABLE internal_cluster_members_new (
 );
 
 INSERT INTO internal_cluster_members_new SELECT id,name,address,certificate,1,(schema-1),heartbeat,role FROM internal_cluster_members;
+
```

Comment:
Ah ok, ignore my comment from before, you have added the triggers in a follow up commit. I got confused by the code comment "Also create some triggers ...".